src/Security/EditableRolesBuilder.php

```diff
@@ -42,23 +39,22 @@ class EditableRolesBuilder
     /**
      * @param array<string, array<string>> $rolesHierarchy
      */
-    public function __construct(TokenStorageInterface $tokenStorage, AuthorizationCheckerInterface $authorizationChecker, Pool $pool, SonataConfiguration $configuration, array $rolesHierarchy = [])
-    {
+    public function __construct(
+        TokenStorageInterface $tokenStorage,
+        AuthorizationCheckerInterface $authorizationChecker,
+        Pool $pool,
+        SonataConfiguration $configuration,
+        ?TranslatorInterface $translator = null,
```

Comment:
Translator can be removed isnt it ? It seems not used

Reply:
Is it used, check below.

Reply:
Indeed, on the last lines.

Test need to be updated and this should be ok.

Reply:
Now, they should be okay.